business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -4,18 +4,18 @@
 
   <parent>
     <groupId>org.kie</groupId>
-    <artifactId>kie-wb-tests</artifactId>
+    <artifactId>business-central-tests</artifactId>
     <version>7.15.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>kie-wb-tests-gui</artifactId>
+  <artifactId>business-central-tests-gui</artifactId>
 
   <name>KIE (Drools) Workbench Tests :: GUI</name>
-  <description>GUI tests for KIE Workbench and KIE Drools Workbench</description>
+  <description>GUI tests for Business Central and KIE Drools Workbench</description>
 
   <properties>
-    <app.name>kie-wb</app.name><!-- for tests to know if they are running against kie-wb or kie-drools-wb -->
-    <selenium.homepage.loading.timeout.seconds>30</selenium.homepage.loading.timeout.seconds>
+    <app.name>kie-wb</app.name><!-- for tests to know if they are running against business-central or kie-drools-wb -->
```

Comment:
This should be made consistent with the rest of the renaming:
`<app.name>kie-wb</app.name>` -> `<app.name>business-central</app.name>`
Also kie-drools-wb should be removed from the comment above.